linera-core/src/node.rs

```diff
@@ -97,9 +100,9 @@ pub trait ValidatorNode {
 }
 
 /// Turn an address into a validator node.
-#[cfg_attr(not(web), trait_variant::make(Send))]
+#[cfg_attr(not(web), trait_variant::make(Send + Sync))]
```

Comment:
I'm a little surprised the `not(web)` here is necessary.  Does it fail to compile without?

Reply:
I haven't tried that, since it was already there. I didn't want to go too far beyond the client map removal in this PR.